app/views/support_interface/application_forms/applicant_details/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :browser_title, title_with_error_prefix('Edit applicant details', @details.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(support_interface_application_form_path(@details.application_form)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @details, url: support_interface_application_form_update_applicant_details_path do |f| %>
+      <%= f.govuk_error_summary %>
+
+      <%= f.govuk_fieldset legend: { text: 'Edit applicant details', size: 'l' } do %>
+        <%= f.govuk_text_field :phone_number, label: { text: 'Phone number' } %>
+      <% end %>
+
```

Comment:
So I don't forget: we could add the audit comment field here where the support user can paste in a link to Zendesk, so that we have context on why the change was made.

Reply:
Very good idea, will add!

Reply:
<img width="1063" alt="Screenshot 2020-11-18 at 10 45 00" src="https://user-images.githubusercontent.com/642279/99520547-23a2d900-298b-11eb-81c9-b4297a1f7025.png">